astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -37,8 +37,8 @@
 from .cirs import CIRS
 from .itrs import ITRS
 from .hcrs import HCRS
-from .ecliptic import (GeocentricTrueEcliptic, BarycentricTrueEcliptic,
-                       HeliocentricTrueEcliptic, BaseEclipticFrame)
+from .ecliptic import (GeocentricMeanEcliptic, BarycentricMeanEcliptic,
```

Comment:
Not sure if I should bring here the `*TrueEcliptic` or the `*CorrectTrueEcliptic` ones.